packages/icons/src/general/IconSnooze.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import withIcon, { Props } from '../withIcon';
+
+function IconSnooze(props: Props) {
```

Comment:
this looks like it needs to be run through prettier.

from your lunar directory, in the terminal, you'll need to run:

```
npm run prettier packages/icons/src/general/IconSnooze.tsx
```